schema/deploy/type_ciip_application_status.sql

```diff
@@ -0,0 +1,8 @@
+-- Deploy ggircs-portal:type_ciip_application_status to pg
+-- requires: schema_ggircs_portal
+
+begin;
+
+create type ggircs_portal.ciip_application_status as enum ('pending', 'approved', 'rejected', 'draft');
```

Comment:
Maybe we need to add a "requested changes" and "changes submitted" values here, to match what is in the review status type.